src/gui/mzroll/alignmentvizallgroupswidget.cpp

```diff
@@ -1,9 +1,27 @@
 #include "alignmentvizallgroupswidget.h"
+#include <QDockWidget>
+#include <QToolBar>
 
 using namespace std;
 
-AlignmentVizAllGroupsWidget::AlignmentVizAllGroupsWidget(MainWindow* mw) {
-    this->_mw = mw;
+AlignmentVizAllGroupsWidget::AlignmentVizAllGroupsWidget(MainWindow* mw, QDockWidget* dockWidget):
```

Comment:
This plot is not visible till alignment is run, even though there is no dependence on the algorithm. Is the plotting function just being called after alignment? Should be called as soon as the widget opens, ideally.

Reply:
I think the Sample rt widget has the same problem but I haven't looked at the code to know if it is independent of the algorithm. I'm guessing it is. In which case, it should be plotted even without alignment.